bodhi/server/buildsys.py

```diff
@@ -78,6 +78,8 @@ class DevBuildsys(Buildsystem):
     __added__ = []
     __tagged__ = {}
     __rpms__ = []
+    multicall = False
+    multicallresult = []
```

Comment:
Since these are class variables, all the places that alter them will alter them across all instances. I suspect that is probably not what you want. I recommend instead setting these in an ```__init__()``` method like this instead:

```
self.multicall = False
self.multicallresult = []
```

That way each instance of DevBuildSys will have independent data on these variables.

Or was it intentional that these are class variables?